prisma/schema.prisma

```diff
@@ -4,70 +4,98 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = "postgres://fndnziax:YEbewgjpbkceCXlmN46uXIYFuMPrkpbs@tyke.db.elephantsql.com/fndnziax"
+  url      = "postgresql://postgres:AnDo45THu9%3F@database-1.cjquiugk0zdh.us-east-1.rds.amazonaws.com:5432"
 }
 
 model Business {
-  id                  Int             @map("BusinessID")
-  UserID              Int?
-  name                String
-  sustainabilityScore Float           @map("FutureproofRating")
-  customerScore       Float           @map("UserRating")
+  id                  Int           @unique @id @default(autoincrement()) @map("BusinessID")
+  owner               User          @relation(fields: [userId], references: [id])
+  locations           Location[]    @relation("Locations")
+  primaryLocation     Location?     @relation("PrimaryLocation")
+  name                String        @unique
+  sustainabilityScore Float         @map("FutureproofRating")
+  customerScore       Float         @map("UserRating")
   ProfileText         String?
+  comments            Comment[]
+  reviews             UserReview[]
+  createdAt           DateTime
+  updatedAt           DateTime
   ProfilePicture      Bytes?
   type                BusinessType? @map("BusinessType")
 
-  @@id([id, name])
+
+  userId     Int
+  locationId Int? @unique
+  // @@id([id, name])
 }
 
 model Comment {
-  CommentID  Int      @id
-  BusinessID Int
-  LocationID Int
-  UserID     Int
-  text       String   @map("CommentText")
-  createdAt  DateTime @map("DateCreated") @db.Date
+  id        Int       @id @unique @default(autoincrement())
+  location  Location  @relation(fields: [locationId], references: [id])
+  user      User      @relation(fields: [userId], references: [id])
+  text      String    @map("CommentText")
+  createdAt DateTime  @map("DateCreated") @db.Date
+  updatedAt DateTime  @db.Date
+  // business  Business? @relation(fields: [businessId], references: [id])
+
+  // businessId   Int?
```

Comment:
Why are these commented out?

Reply:
In case the edits were wrong and we need to return to previous code. Same for below comment. Not necessary to keep but useful.